### ISSUE TITLE
Disable EFI ?

### ISSUE THREAD

**Issue by Shadok** [livecd-tools/livecd-tools#261]
Hello,

Is there a way to disable efi while creating an livecd iso ?

Altough i don't specify anything about efi and don't install any related package, i still have this during the process :
```
Missing EFI file (/boot/efi/EFI/*/shimx64.efi)
Missing EFI file (/boot/efi/EFI/*/gcdx64.efi)
Failed to copy EFI files, no EFI Support will be included.
Missing initial EFI bootloader, skipping efiboot.img creation.
usage: mkefiboot [-h] [--debug] [-d] [-a] [-l LABEL] [-i ICONFILE]
                 [-n DISKNAME] [-p PRODUCT]
                 EFIBOOTDIR OUTPUTFILE
mkefiboot: error: /opt/livecd/tmp/imgcreate-g5g5u6me/iso-lmxp5pjg/EFI/BOOT is not a directory
```

I don't use efi (our servers don't support it, so our os masters doesn't either).

Thanks

**Comment by microfire21** [comment]
so been working on a little project that I really want fedora 37 to work with. I have installed newest checkout 4acb0c9515783446c3ad87bf29ec049ed835f788. No matter what options I throw at it I always get error 1 at line 2028.

`[live@fedora ~]$ sudo livecd-iso-to-disk --format ext4 --overlay-size-mb 4096 --efi Fedora-Workstation-Live-x86_64-37-1.7.iso  /dev/sdb

Source image is '/home/micro/Fedora-Workstation-Live-x86_64-37-1.7.iso'
Verifying image...
/home/micro/Fedora-Workstation-Live-x86_64-37-1.7.iso:   0f0a539212821d61fcedfda0b8409869
Fragment sums: b7843fb48cd5d5c9f8be2416df66448c25bd5d62d7877a937e6f8c8d6dfc
Fragment count: 20
Supported ISO: no
Press [Esc] to abort check.
Checking: 100.0%

The media check is complete, the result is: PASS.

It is OK to use this media.
Error 1 detected at line 2028 /usr/bin/livecd-iso-to-disk
Error 1 detected at line 2028 /usr/bin/livecd-iso-to-disk
`

**Comment by sobjerke** [comment]
The missing files are in grub2-efi-x64-cdboot and shim-x64. These RPMs should not harm a non-EFI server.

**Comment by Shadok** [comment]
Indeed, the iso works, the point is just to avoid the non-blocking error in the output.
